pkg/vpcmodel/nodesExplainability.go

```diff
@@ -213,31 +256,32 @@ func (c *VPCConfig) getRulesOfConnection(src, dst Node, conn *common.ConnectionS
 }
 
 // node is from getCidrExternalNodes, thus there is a node in VPCConfig that either equal to or contains it.
-func (c *VPCConfig) getContainingConfigNode(node Node) (Node, error) {
+func (c *VPCConfig) getContainingConfigNode(node Node) Node {
 	if node.IsInternal() { // node is not external - nothing to do
-		return node, nil
+		return node
 	}
 	nodeIPBlock := common.NewIPBlockFromCidr(node.Cidr())
-	if nodeIPBlock == nil { // string cidr does not represent a legal cidr
-		return nil, fmt.Errorf("could not find IP block of external node %v", node.Name())
+	if nodeIPBlock == nil { // string cidr does not represent a legal cidr, would be handled earlier
```

Comment:
doesn't this mean the node is invalid and we should stop processing?

Reply:
This should be part of the input checks.
Given that the input is legal we should never get here.
Do you think we should still add an error here and stop?

Reply:
I think it's better to be explicit and add an error here.
Also the other function (`func (v *VPCConnectivity) getConnection`) that calls `getContainingConfigNode()` does return an error in such a case.

Reply:
done